templates/CRM/Admin/Form/ScheduleReminders.tpl

```diff
@@ -280,6 +276,15 @@
           showSaveUpdateChkBox('SMS');
         }
       }
+	  
+	  //CRM-14097 - When un-checking the "send email" box, the email fields do not get hidden
+	  cj(function ( $ ) {
+		$(document).ready(function(){
+			$('body').on('change', '.crm-scheduleReminder-form-block-active #send_email', function () { 
+				$('.crm-collapsible').toggle();
```

Comment:
Instead of attaching this event to the body, I would attach it to the form. The variable `$form` is already available to you.
Also, `$('.crm-collapsible').toggle();` is not nearly specific enough - which `.crm-collapsible` element, exactly, do you wish to toggle? At the very least use the `$form` variable as the 2nd argument to this to limit the scope.